dns_test.go

```diff
@@ -405,3 +407,41 @@ func TestMsgPackBuffer(t *testing.T) {
 		}
 	}
 }
+
+// Make sure we can decode a TKEY packet from the string, modify the RR, and then pack it again.
+func TestTKEY (t *testing.T) {
```

Comment:
Thanks.

This needs gofmt

Reply:
whoops, my bad - new project was setup right in my IDE.  will fix.